### PR TITLE
feat: add lago

### LINKED ISSUE #739: App Suggestion: GetLago
Can Any one Support Adding the below App to Caprover One click apps?

Open Source Billing API for Product-Led SaaS

https://github.com/getlago/lago

Docker compose file here:
https://github.com/getlago/lago/blob/main/docker-compose.arm64.yml

### PR DESCRIPTION
**Pull request by ronaldloyko**
This pull request adds [Lago](https://www.getlago.com/).

## Checks

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

## References

- closes #739

### CODE REVIEW

**Comment by githubsaturn** [comment]
Looks good! Thanks!

**Comment by TarekMahran** [comment]
Hi githubsaturn,
Many Thanks for adding lago to one click apps. I tried it and it works fine. Only one issue appears when I tried to activate https for Lago front end url:
**1107 : Verification Failed.**
Waiting your support to solve this issue.

Regards,

**Comment by githubsaturn** [comment]
First off, it's actually @ronaldloyko who added this new app :)

With regards to the verification failure, it is not the app issue. You can look at the docs at caprover.com to see the actions that you can take at this point

**Comment by ronaldloyko** [comment]
Hi @TarekMahran 
This will solve your issue: https://caprover.com/docs/troubleshooting.html#domain-verification-failed---error-1107

```
echo  "{\"skipVerifyingDomains\":\"true\"}" >  /captain/data/config-override.json
docker service update captain-captain --force
```

**Comment by ronaldloyko** [comment]
You were few seconds faster @githubsaturn :smile:

**Comment by TarekMahran** [comment]
Thanks @ronaldloyko and @githubsaturn for your prompt support. I will try your solution.

**Comment by TarekMahran** [comment]
Hi @ronaldloyko, I tried your solution, and works fine like a charm.

File: public/v4/apps/lago.yml [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+            ENCRYPTION_PRIMARY_KEY: $$cap_LAGO_API_ENCRYPTION_PRIMARY_KEY
+            ENCRYPTION_DETERMINISTIC_KEY: $$cap_LAGO_API_ENCRYPTION_DETERMINISTIC_KEY
+            ENCRYPTION_KEY_DERIVATION_SALT: $$cap_LAGO_API_ENCRYPTION_KEY_DERIVATION_SALT
+            LAGO_USE_AWS_S3: $$cap_LAGO_API_USE_AWS_S3
+            LAGO_AWS_S3_ACCESS_KEY_ID: $$cap_LAGO_API_AWS_S3_ACCESS_KEY_ID
+            LAGO_AWS_S3_SECRET_ACCESS_KEY: $$cap_LAGO_API_AWS_S3_SECRET_ACCESS_KEY
+            LAGO_AWS_S3_REGION: $$cap_LAGO_API_AWS_S3_REGION
+            LAGO_AWS_S3_BUCKET: $$cap_LAGO_API_AWS_S3_BUCKET
+            LAGO_AWS_S3_ENDPOINT: $$cap_LAGO_API_AWS_S3_ENDPOINT
+            LAGO_PDF_URL: http://srv-captain--$$cap_appname-pdf
```
Comment (aqidd) [inline public/v4/apps/lago.yml:42]: this should use port 3000 instead of 80

**Pull request merged**